clients/java/src/test/java/marquez/client/MarquezClientTest.java

```diff
@@ -955,120 +983,21 @@ public void testCreateTag() throws Exception {
-            new ColumnLineageNodeData(
-                NAMESPACE_NAME,
-                DB_TABLE_NAME,
-                FIELD_NAME,
-                "String",
-                Collections.singletonList(
-                    new ColumnLineageInputField(
-                        "namespace",
-                        "inDataset",
-                        "some-col1",
-                        "transformationDescription",
-                        "transformationType"))),
-            ImmutableSet.of(
-                Edge.of(
-                    NodeId.of(DATASET_FIELD_ID),
-                    NodeId.of(new DatasetFieldId("namespace", "inDataset", "some-col1")))),
-            ImmutableSet.of(
-                Edge.of(
-                    NodeId.of(new DatasetFieldId("namespace", "outDataset", "some-col2")),
-                    NodeId.of(DATASET_FIELD_ID))));
-    MarquezClient.Lineage lineage = new MarquezClient.Lineage(ImmutableSet.of(node));
-    String lineageJson = lineage.toJson();
-    when(http.get(
-            buildUrlFor(
-                "/column-lineage?nodeId=datasetField%3Anamespace%3Adataset%3Asome-col1&depth=20&withDownstream=false")))
-        .thenReturn(lineageJson);
-
-    Node retrievedNode =
-        client.getColumnLineageByDataset("namespace", "dataset", "some-col1").getGraph().stream()
-            .findAny()
-            .get();
-    assertThat(retrievedNode).isEqualTo(node);
-  }
-
-  @Test
-  public void testGetColumnLineageByJob() throws Exception {
-    Node node =
-        new Node(
-            NodeId.of(DATASET_FIELD_ID),
-            NodeType.DATASET_FIELD,
-            new ColumnLineageNodeData(
-                NAMESPACE_NAME,
-                DB_TABLE_NAME,
-                FIELD_NAME,
-                "String",
-                Collections.singletonList(
-                    new ColumnLineageInputField(
-                        "namespace",
-                        "inDataset",
-                        "some-col1",
-                        "transformationDescription",
-                        "transformationType"))),
-            ImmutableSet.of(
-                Edge.of(
-                    NodeId.of(DATASET_FIELD_ID),
-                    NodeId.of(new DatasetFieldId("namespace", "inDataset", "some-col1")))),
-            ImmutableSet.of(
-                Edge.of(
-                    NodeId.of(new DatasetFieldId("namespace", "outDataset", "some-col2")),
-                    NodeId.of(DATASET_FIELD_ID))));
-    MarquezClient.Lineage lineage = new MarquezClient.Lineage(ImmutableSet.of(node));
-    String lineageJson = lineage.toJson();
-    when(http.get(
-            buildUrlFor(
-                "/column-lineage?nodeId=job%3Anamespace%3Ajob&depth=20&withDownstream=false")))
-        .thenReturn(lineageJson);
-
-    Node retrievedNode =
-        client.getColumnLineageByJob("namespace", "job").getGraph().stream().findAny().get();
-    assertThat(retrievedNode).isEqualTo(node);
```

Comment:
Nice bit of test code simplicity ❤️

Reply:
Thanks. I've added note on that to `changelog`.